engine/src/main/java/org/terasology/rendering/dag/nodes/ShadowMapNode.java

```diff
@@ -48,6 +47,7 @@
  * - https://docs.google.com/drawings/d/13I0GM9jDFlZv1vNrUPlQuBbaF86RPRNpVfn5q8Wj2lc/edit?usp=sharing
  */
 public class ShadowMapNode extends AbstractNode {
+    private static final String SHADOW_MAP_FBO_NAME = "sceneShadowMap";
```

Comment:
Thank you. In any case the type of the variable gives it away that it cannot be the FBO itself. =)